### ISSUE TITLE
Why do we need the `process.context_switches`?

### ISSUE THREAD

**Issue by Yun-Ting** [open-telemetry/opentelemetry-specification#2929]
**What are you trying to achieve?**

Understanding the necessity of having `process.context_switches` metric.

What did you expect to see?

Process context switch is tracking how multiple processes share a single CPU and how the CPU multi-task.
I am not sure how would having the `process.context_switches` benefit the metric consumer in understanding the process behavior/performance.
I propose to remove this metric from the spec.

**Comment by evan-bradley** [comment]
The `process.context_switches` metric can be used to help determine whether a process is IO-bound or CPU-bound using the `type` attribute to compare the number of voluntary and involuntary context switches. The metric is [optionally emitted](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md) by the Collector's host metrics receiver.

**Comment by Yun-Ting** [comment]
> The `process.context_switches` metric can be used to help determine whether a process is IO-bound or CPU-bound using the `type` attribute to compare the number of voluntary and involuntary context switches. The metric is [optionally emitted](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md) by the Collector's host metrics receiver.

Yes, I think collector is a more suitable place to track the process.context_switch rather than having it in the process instrumentation space.

**Comment by ocelotl** [comment]
I asked Claude to review this issue and Claude suggests it be closed with this explanation:

> A question/removal-proposal for process.context_switches. Maintainer @evan-bradley explained its use (IO- vs CPU-bound via voluntary/involuntary type attr) and that semconv inclusion does not require reporting; his clarifying comment got 3 thumbs-up and the removal proposal was effectively declined. Question answered in-thread; metric now defined in the open-telemetry/semantic-conventions repo, so any further discussion belongs there, not here.

To guard against a mistaken suggestion, Claude then independently re-verified this conclusion:

> Removal-proposal question fully answered by maintainer (3 thumbs-up, no counter), reporter agreed the metric belongs in Collector/semconv; dead since Nov 2022 and process.context_switches now lives in the semantic-conventions repo, so nothing actionable remains here.

Claude evaluated this issue with medium confidence that it can be closed.

Claude has classified this issue as an answered question.

The intention of this comment is to make it easier for a human being to review this issue before making a decision.
